Cargo.toml

```diff
@@ -9,5 +9,10 @@ homepage = "https://github.com/conradkleinespel/rustastic-password"
 readme = "README.md"
 keywords = ["read", "password"]
 
-[dependencies]
```

Comment:
The removal of these two lines break builds on UNIX. Is there a particular reason you removed these?